packages/modules/notification/src/loaders/providers.ts

```diff
@@ -0,0 +1,116 @@
+import { moduleProviderLoader } from "@medusajs/modules-sdk"
```

Comment:
note: We will have to rework this as we have it in 3 places now 👍

Reply:
@adrien2p oh there is so much to improve around loading providers, but I didn't want to do it as part of this PR

Reply:
yes definately, it is more a note that a suggestion :)